### ISSUE TITLE
Bug: Slicing Failed Model Too Small But Worked Fine on Older Version

### ISSUE THREAD

**Issue by ezraincorvaia** [Snapmaker/Luban#1604]
# 🐞 bug report
When trying to slice a model that is fairly large and I have printed before, I get a slicing error that it is too small. Image of error attached
![Screenshot (203)](https://user-images.githubusercontent.com/73658230/171215880-e6e06f32-5e5a-416a-8d40-0dd1246b6126.png)

### Affected Version(s)
This issue was caused by Version 4.3

### Is this a regression? (optional)
The previous version in which this bug was not present was 4.2.3

### To Reproduce
1. Go to new 3D print project
2. Add model to be sliced
3. Pick any 3D Print Profile settings
4. Click Generate G-Code to slice and show preview
5. It loads for a moment and then the Slicing Failed (Slicing Failed because model is too small)

Expected behavior
When I hit the Generate G-Code button I expect, the software to think for a moment and then show me the preview of the model I am trying to slice with

## 🌍  Your Environment

**Platform:**
<pre><code>
- Operating System: Windows 10 x64
- Printer: Snapmaker 2.0 A150

</code></pre>

**Comment by ezraincorvaia** [comment]
Incorvaia,Ezra has shared a OneDrive for Business file with you. To view it, click the link below.
<https://drexel0-my.sharepoint.com/personal/lmi32_drexel_edu/Documents/Not%20School/Books/Attachments/Tower_Tray_Knurl.obj>
[https://r1.res.office365.com/owa/prem/images/dc-generic_20.png]<https://drexel0-my.sharepoint.com/personal/lmi32_drexel_edu/Documents/Not%20School/Books/Attachments/Tower_Tray_Knurl.obj>
Tower_Tray_Knurl.obj<https://drexel0-my.sharepoint.com/personal/lmi32_drexel_edu/Documents/Not%20School/Books/Attachments/Tower_Tray_Knurl.obj>

Of course!

Ezra Incorvaia
PGP: He/Him/His
Dragons Remember Student Researcher
Animation & VFX Senior
________________________________

**Comment by ezraincorvaia** [comment]
To view Tower_Tray_Knurl.obj, sign in<https://drexel0-my.sharepoint.com/personal/lmi32_drexel_edu/_layouts/15/acceptinvite.aspx?invitation=%7B6547BF52%2DEFB0%2D4E1C%2DB80F%2D39422E7B1946%7D&listId=1c138326%2D4dfb%2D450e%2D8479%2Db4d1e3722272&itemId=91522df7%2D9edf%2D4fe6%2D8298%2D273a5cf72fab> or create an account.

**Comment by jane-rose** [comment]
Does't get invitation? It shows `We're sorry, but `email` can't be found in the drexel0-my.sharepoint.com directory. Please try again later, while we try to automatically fix this for you.` Leave my email,  [jt@snapmaker.com](jt@snapmaker.com)

**Comment by olive007** [comment]
I have the same issue, right now...
Is this solve ?

**Comment by urbanadventurer** [comment]
I found a workaround for SnapMaker Luban Version 4.3.2 (4.3.2) on my Mac.

1. Close Snapmaker Luban

2. Delete the local settings

You can use this command from the terminal or delete this folder in Finder
```
rm -Rf ~/Library/ApplicationSupport/snapmaker-luban
```

4. Restart Snapmaker Luban

**Comment by GUNFAI** [comment]
Same problem on Windows10. Upgraded Luban but the problem continued.
I got the error on all models, even those I pronted before. 
Solution; close Luban,
delete all cache files under users/[admin]/appdata/roaming/snapmaker-luban/cach
And then start Luban

**Comment by AdrianQuintanilla** [comment]
> 

Yeah, I can't slice at all. Not sure what's wrong. Machine: Windows 10, 64bit. 
Luban 4.6.3

I do not see any appdata folder when I go to users/[admin]. 
![Slice Failed](https://user-images.githubusercontent.com/44623160/224515569-64b3dde3-74d0-4a19-a994-caa8d5d78cf3.PNG)

Is there another way of getting to the snap-maker-luban/cach ?
